app/src/main/java/com/babylon/wallet/android/domain/usecases/ResolveComponentAddressesUseCase.kt

```diff
@@ -29,7 +30,7 @@ class ResolveComponentAddressesUseCase @Inject constructor(
                 isRefreshing = true
             ).mapCatching { dApps ->
                 val dApp = dApps.first()
-                componentAddress to dApp
+                componentAddress to dApp.takeIf { it.claimedEntities.contains(componentAddress.string) }
```

Comment:
Regarding this, this is correct now but do you remember what problems #782 was trying to solve?

Reply:
I think this line change was done by mistake, main takeaway in previous PR was to use `dapp_definition` instead of `dapp_definitions`